### PR TITLE
[API-Draft][Connector] New file sink connector

### PR DESCRIPTION
**Pull request by EricJoy2048**
feature https://github.com/apache/incubator-seatunnel/issues/1946

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -614,43 +615,23 @@
...
+                <scope>test</scope>
             </dependency>
             <dependency>
-                <groupId>org.apache.hadoop</groupId>
-                <artifactId>hadoop-client</artifactId>
-                <version>${hadoop2.version}</version>
+                <groupId>org.powermock</groupId>
+                <artifactId>powermock-api-mockito2</artifactId>
+                <version>2.0.9</version>
+                <scope>test</scope>
```
Comment (CalvinKirs) [inline pom.xml]: Please declare version in `<properties>`

File: seatunnel-connectors/seatunnel-connectors-seatunnel/seatunnel-connector-hadoop-shade/seatunnel-hadoop2-shade/pom.xml [hunk 2]
```diff
@@ -0,0 +1,131 @@
...
+            <groupId>org.apache.hadoop</groupId>
+            <artifactId>hadoop-hdfs</artifactId>
+            <version>${hadoop2.version}</version>
+        </dependency>
+        <dependency>
+            <groupId>org.apache.hadoop</groupId>
+            <artifactId>hadoop-client</artifactId>
+            <version>${hadoop2.version}</version>
+        </dependency>
+    </dependencies>
```
Comment (CalvinKirs) [inline seatunnel-connectors/seatunnel-connectors-seatunnel/seatunnel-connector-hadoop-shade/seatunnel-hadoop2-shade/pom.xml]: Note if this shade is still needed, and if so, does it actually solve the problem. In my local test, usually some other classes he carries such as `fastxml`,`jackson`...etc will cause conflicts.

Comment (EricJoy2048) [reply seatunnel-connectors/seatunnel-connectors-seatunnel/seatunnel-connector-hadoop-shade/seatunnel-hadoop2-shade/pom.xml]: No, We don't need this. I removed it and I will commit it when a finish the test about it.

**Comment by EricJoy2048** [comment]
Hi gays, The name of our connector module is too long. I think we should optimize the connector name.

image

**Pull request closed without merging**